test/libsolidity/syntaxTests/comments/multiline_unicode_direction_override_1.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.7.0;
```

Comment:
I thought we were not using version pragmas and license comments in tests? Updating versions for 0.8.0 won't be fun :)

Reply:
I think this makes sense and removed those from tests